src/playercards/PlayerCardPanel.ttslua

```diff
@@ -361,6 +382,35 @@ function updateStarterModeButtons()
   createInvestigatorModeButtons()
 end
 
+-- Clears the table and updates positions based on scale.  Should be called before ANY card
+-- placement
+function prepareToPlaceCards()
+  deleteAll()
+  scalePositions()
+end
+
+-- Updates the positions based on the current object scale to ensure the relative layout functions
+-- properly at different scales.
+function scalePositions()
+  -- Assume scaling is consistent in X and Z dimensions
+  local scale = 1 / self.getScale().x
+  startPositions = { }
+  for key, pos in pairs(START_POSITIONS) do
```

Comment:
It could be.  I actually kept the names similar intentionally so that the connection between them would be clear.  The constants should never be used in code, hopefully made that clear enough with the comments.  The nature of the global vs. local distances will also make it REALLY obvious if someone uses the wrong one, so it should be safe.